### PR TITLE
FIX-PS5Controller

### PR DESCRIPTION
**Pull request by NadaVibe**

### CODE REVIEW

File: src/joysticks/ConsoleController.java [hunk 1]
```diff
@@ -0,0 +1,448 @@
...
+        this.stickRight = stickRight;
+        this.axisLeftX = axisLeftX;
+
+        this.axisLeftY = axisLeftY;
+        this.axisRightX = axisRightX;
+        this.axisRightY = axisRightY;
+        this.leftTrigger = leftTrigger;
+        this.rightTrigger = rightTrigger;
+        this.bumperRight = bumperRight;
+        this.bumperLeft = bumperLeft;
```
Comment (YuvalGur135) [inline src/joysticks/ConsoleController.java:49]: please reorgenize the order of those params

File: src/joysticks/ConsoleController.java [hunk 2]
```diff
@@ -0,0 +1,448 @@
...
+     * @return The axis value.
+     */
+    public double getRightY() {
+        return getRawAxis(axisRightY);
+    }
+
+    /**
+     * Get the port of the controller.
+     */
+    public int getPort(){
```
Comment (YuvalGur135) [inline src/joysticks/ConsoleController.java:91]: this func should probobly be first.

Comment (YuvalGur135) [inline src/joysticks/ConsoleController.java:91]: space between -> () {

File: src/joysticks/ConsoleController.java [hunk 3]
```diff
@@ -0,0 +1,448 @@
...
+    public int getRightTrigger() {
+        return rightTrigger;
+    }
+
+    public int getBumperRight() {
+        return bumperRight;
+    }
+
+    public int getBumperLeft() {
+        return bumperLeft;
```
Comment (YuvalGur135) [inline src/joysticks/ConsoleController.java:446]: if u r very very very bored try to put in the same order as of the - raw butt funcs

File: src/joysticks/OnyxXboxController.java [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+import edu.wpi.first.hal.FRCNetComm;
+import edu.wpi.first.hal.HAL;
+
+public class OnyxXboxController extends ConsoleController {
+
+    /**
+     * Represents a digital button on an XboxController.
+     */
+    public enum Button {
+        kBumperLeft(5),
```
Comment (YuvalGur135) [inline src/joysticks/OnyxXboxController.java:12]: blank line here

File: src/joysticks/OnyxXboxController.java [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+        kBumperRight(6),
+        kStickLeft(9),
+        kStickRight(10),
+        kA(1),
+        kB(2),
+        kX(3),
+        kY(4),
+        kBack(7),
+        kStart(8);
```
Comment (YuvalGur135) [inline src/joysticks/OnyxXboxController.java:21]: right order of numbers plz

File: src/joysticks/OnyxXboxController.java [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+        Button(int value) {
+            this.value = value;
+        }
+    }
+
+    /**
+     * Represents an axis on an XboxController.
+     */
+    public enum Axis {
+        kLeftX(0),
```
Comment (YuvalGur135) [inline src/joysticks/OnyxXboxController.java:35]: blank line

File: src/joysticks/OnyxXboxController.java [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+        kRightX(4),
+        kLeftY(1),
+        kRightY(5),
+        kLeftTrigger(2),
+        kRightTrigger(3);
```
Comment (YuvalGur135) [inline src/joysticks/OnyxXboxController.java:40]: right order of numbers plz

File: src/joysticks/PlayStation5Controller.java [hunk 8]
```diff
@@ -0,0 +1,73 @@
...
+import edu.wpi.first.hal.FRCNetComm.tResourceType;
+import edu.wpi.first.hal.HAL;
+
+public class PlayStation5Controller extends ConsoleController {
+
+    /**
+     * Represents a digital button on an PS5 Controller.
+     */
+    public enum Button {
+        kSquare(1),
```
Comment (YuvalGur135) [inline src/joysticks/PlayStation5Controller.java:12]: blank linenenenenenen

File: src/joysticks/PlayStation5Controller.java [hunk 9]
```diff
@@ -0,0 +1,73 @@
...
+        Button(int value) {
+            this.value = value;
+        }
+    }
+
+    /**
+     * Represents an axis on an PS5 Controller.
+     */
+    public enum Axis {
+        kLeftX(0),
```
Comment (YuvalGur135) [inline src/joysticks/PlayStation5Controller.java:37]: yoohooo blank line!!! wH3rE ^Re y0U????

**Review by YuvalGur135 — CHANGES_REQUESTED** [review]

**Review by nimrod46 — APPROVED** [review]

**Pull request merged**